src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -155,9 +160,12 @@ private ScheduledInterviewRequest makeScheduledInterviewRequest(
             .get(scheduledInterview.intervieweeId())
             .map(Person::firstName)
             .orElse("Nonexistent User");
+    String role = getUserRole(scheduledInterview, userId);
+    boolean hasPassed =
```

Comment:
l feel like this name is misleading. lt's not whether the interview has passed, it's more of whether it has started. Please change accordingly.

Reply:
Done